pkg/steps/pod.go

```diff
@@ -161,10 +162,15 @@ func (s *podStep) Objects() []ctrlruntimeclient.Object {
 	return s.client.Objects()
 }
 
-func (s *podStep) IsMultiArch() bool {
-	return s.config.NodeArchitecture != "" && s.config.NodeArchitecture != api.NodeArchitectureAMD64
+func (s *podStep) ResolveMultiArch() sets.Set[string] {
```

Comment:
Here is the fix for https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/56924/pull-ci-openshift-release-master-core-valid/1837196930166296576#1:build-log.txt%3A12

Reply:
vs https://github.com/openshift/ci-tools/pull/4334/files#diff-c7d8e7bf69599502da04b57d8e3b1100a7239e6446109540aa8fd9afdda45842R165-R166